WordPress/gutenberg
# [RNMobile] Fix notice size overflow

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3411





## Description


Related to https://github.com/WordPress/gutenberg/pull/31012/files#r617418413.

### Problem

The native version of the notice component doesn't handle properly the size overflow when the text is larger than one line. You can see in the attached screenshots how it looks in the current version.

One line|Two lines|Three lines|Overflow
--|--|--|--
<img src=https://user-images.githubusercontent.com/14905380/115706418-e98c9780-a36d-11eb-8793-be29d3cf5b80.png>|<img src=https://user-images.githubusercontent.com/14905380/115706423-eb565b00-a36d-11eb-9cf4-d8f672afc496.png>|<img src=https://user-images.githubusercontent.com/14905380/115706428-ec878800-a36d-11eb-82cf-816a94bdf48b.png>|<img src=https://user-images.githubusercontent.com/14905380/115706432-edb8b500-a36d-11eb-9902-62d325375a5e.png>

### Solution

I basically updated the styles of this component so the view now fits the text size. Additionally, I set the maximum number of lines for the text to three, this way we prevent that a notice gets too big.

## How has this been tested?




For testing this, manual code changes are required to test different text sizes. I tested this by modifying the text of the notice that is triggered when copying a block:

1. Go to this [line](https://github.com/WordPress/gutenberg/blob/755a2272cf0985e5b56978cb506fd6522353c6a0/packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js#L135)
2. Change the text (it would be nice to test with different text sizes like 2/3/4 lines)
3. Select any block
4. Tap on the three dots button to open the block actions menu
5. Tap on "Copy block" (this will display a notice)
6. Observe that the content of the notice is displayed properly

## Screenshots 

One line|Two lines|Three lines|Overflow
--|--|--|--
<img src=https://user-images.githubusercontent.com/14905380/115706562-13de5500-a36e-11eb-9b88-52ce1dab05e0.png>|<img src=https://user-images.githubusercontent.com/14905380/115706569-15a81880-a36e-11eb-8ca8-c1db4d26c718.png>|<img src=https://user-images.githubusercontent.com/14905380/115706575-16d94580-a36e-11eb-8e05-c3e7d9d29bfb.png>|<img src=https://user-images.githubusercontent.com/14905380/115706582-180a7280-a36e-11eb-9adf-e917f0208274.png>


## Types of changes




Bug fix

## Checklist:
